### PR TITLE
LPD-33171 | Master

### PR DESCRIPTION
**Pull request by interaminense**
1. Create FF https://liferay.atlassian.net/browse/LPD-33171 https://github.com/liferay-ac/liferay-portal/pull/1744/commits/094ff66bfd3fdc24dda56ab2ba067a03582d3220
2. Create Performance Tab https://liferay.atlassian.net/browse/LPD-33169 https://github.com/liferay-ac/liferay-portal/pull/1744/commits/7a7294c34cdd519a98fe95faa3d005c421089e6f, https://github.com/liferay-ac/liferay-portal/pull/1744/commits/5b3b831f2fe120ad5a3420f8435d81febd7e8ac7, https://github.com/liferay-ac/liferay-portal/pull/1744/commits/487258feb43b0c05803e4c2b35bc3fbfc14c6d47, https://github.com/liferay-ac/liferay-portal/pull/1744/commits/6138f4ba284dffbec5d8ef3c4f683e2a3104d2f8
3. Hide performance icon https://liferay.atlassian.net/browse/LPD-33730 https://github.com/liferay-ac/liferay-portal/pull/1744/commits/74397f42ca2c7492da7e1a1d7f0aca0a00502942

### CODE REVIEW

**Comment by interaminense** [comment]
ci:test:sf

**Comment by interaminense** [comment]
ci:test:relevant

**Comment by interaminense** [comment]
ci:test:sf

File: modules/dxp/apps/analytics/analytics-reports-js-components-web/package.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+	"dependencies": {
+		"react": "16.12.0"
+	},
+	"main": "js/index.js",
+	"name": "analytics-reports-js-components-web",
```
Comment (bryceosterhaus) [inline modules/dxp/apps/analytics/analytics-reports-js-components-web/package.json:6]: nit: rename this to be prefixed with `@liferay/`

File: modules/npmscripts.config.js [hunk 2]
```diff
@@ -540,6 +543,7 @@ module.exports = {
 			'adaptive-media-web',
 			'admin-dxp-theme',
 			'analytics-client-js',
+			'analytics-reports-js-components-web',
```
Comment (bryceosterhaus) [inline modules/npmscripts.config.js:546]: This shouldn't be here, same with the other analytics modules. Everything should ideally have a @liferay/ prefix

File: modules/npmscripts.config.js [hunk 3]
```diff
@@ -155,6 +155,9 @@ module.exports = {
 					'@liferay/template-web': {
 						'/': '*',
 					},
+					'analytics-reports-js-components-web': {
```
Comment (bryceosterhaus) [inline modules/npmscripts.config.js:158]: I think this also needs to be added to build.imports in this config file
Screenshot 2024-08-14 at 1 53 52 PM

**Review by bryceosterhaus — COMMENTED** [review]
I think the biggest thing is needing to add your module to the build.imports config in npmscripts.config.js. See my comment.

Other than that, I think everything is wired correctly

File: modules/dxp/apps/analytics/analytics-reports-js-components-web/package.json (hunk 1 above)
Comment (interaminense) [reply modules/dxp/apps/analytics/analytics-reports-js-components-web/package.json]: Done

File: modules/npmscripts.config.js (hunk 2 above)
Comment (interaminense) [reply modules/npmscripts.config.js]: Done

File: modules/npmscripts.config.js (hunk 3 above)
Comment (interaminense) [reply modules/npmscripts.config.js]: Done

**Comment by interaminense** [comment]
ci:test:sf

**Comment by interaminense** [comment]
Resent https://github.com/liferay-content-management/liferay-portal/pull/5785

**Pull request closed without merging**